tests/test_import.py

```diff
@@ -218,3 +222,50 @@ def test_import_corrupted(fixtures):
                 .dicts()) == [
                     {'has_file': 'M', 'wants_file': 'Y'}
     ]
+
+
+def test_watchdog(fixtures):
+    """Checks that the file system observer triggers imports on new/changed files"""
+    tmpdir = fixtures['root']
+
+    acq_dir = tmpdir.join('12345678T000000Z_inst_zab')
+
+    node = st.StorageNode.get(st.StorageNode.name == 'x')
+
+    from watchdog.observers import Observer
+    obs = Observer()
+    obs.schedule(auto_import.RegisterFile(node),
+                 str(tmpdir),
+                 recursive=True)
+
+    f = acq_dir.join('new_file.log')
+    f.write('')
+
+    # this file is outside any acqs and should be ignored
+    g = tmpdir.join('some_file.log')
+    g.write('Where is my acq?!')
+
+    # nothing happens until the observer is running
+    assert file_copy_count('new_file.log') == 0
+    assert file_copy_count('some_file.log') == 0
+
+    obs.start()
```

Comment:
It's not obvious to me that the watchdog should notice filesystem events before `obs.start` is called. In that case you wouldn't expect the test to pass (maybe MacOS has slightly different behaviour which means it does). Can we shift the test to, create the files before the watchdog starts, test that nothing happens, and then rm them, start the watchdog, recreate them, and test that they do get imported.